### PR TITLE
Ik/update tools and readme

### PR DESCRIPTION
**Pull request by Ilya-Krylov**

### CODE REVIEW

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: README.md [hunk 1]
```diff
@@ -28,148 +28,191 @@ inference.
...
+                           Comma-separated paths to validation annotation files.
+     --val-data-roots VAL_DATA_ROOTS
+                           Comma-separated paths to validation data folders.
+     --load-weights LOAD_WEIGHTS
+                           Load only weights from previously saved checkpoint
+     --save-weights SAVE_WEIGHTS
+                           Location to store wiehgts.
+   ```
+                        
+3. Let's have a look at `ote_train` hyper parmeters help. These parameters are specific all model templates.
```
Comment (druzhkov-paul) [inline README.md:107]:
```suggestion
3. Let's have a look at `ote_train` hyper parameters help. These parameters are specific all model templates.
```

Comment (druzhkov-paul) [reply README.md:107]: `These parameters are specific all model templates.`
Don't quite get this.

File: README.md [hunk 2]
```diff
@@ -28,148 +28,191 @@ inference.
...
+   ```
+   ./external/mmdetection/init_venv.sh det_venv
+   source det_venv/bin/activate
+   pip3 install -e ote_cli/
+   ```
+   
+### OTE CLI tools
+1. Have a look at model templates available for this virtual environment:
+   ```
+   ote_find --root ./external/mmdetection/
```
Comment (druzhkov-paul) [inline README.md:53]: Just an idea. Make `ote` as an entrypoint to all tools. I.e. `ote_find` -> `ote find`.

File: README.md [hunk 3]
```diff
@@ -28,148 +28,191 @@ inference.
...
-    pip3 install -e ote_cli/
-    ```
+2. Let's have a look at `ote_train` help. These parameters are the same for all model templates.
+   ```
+   ote_train ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml --help
+   ```
+   
+   Approximate output:
+   ```
+   ote_train ./external/mmdetection/configs/ote/custom-object-detection/gen3_mobilenetV2_ATSS/template.yaml --help
```
Comment (druzhkov-paul) [inline README.md:81]: Would it make sense to allow the user to provide either the path or the name/id?

**Review by druzhkov-paul — APPROVED** [review]
Some minor comments, feel free to ignore after reading.

File: README.md (hunk 3 above)
Comment (Ilya-Krylov) [reply README.md:81]: Will add next time

**Pull request merged**